ztp/source-crs/extra-manifest/01-container-mount-ns-and-kubelet-conf.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
-  name: container-mount-namespace
+  name: container-mount-namespace-and-kubelet-conf
```

Comment:
Please remove this file and replace it with a symlink to ../MachineConfigContainerMountNS.yaml

Reply:
Done